### DISCUSSION TITLE
Create pole figures from a list of simulated Euler angles

### DISCUSSION THREAD

**Discussion by matheusGariglio** [mtex-toolbox/mtex#1447] [Ask Anything] [answered]
Hi,

I've been doing some crystal plasticity simulations, where I get the three Euler angles for my grains.

I would like to know how can I plot a pole figure from a text file with 3 columns (phi1,Phi,phi2). 

What I tried to do is that I took the text file below (x,y,Euler1,Euler2,Euler3):

0 0 3.311 0.154 3.259
0 0 3.311 0.154 3.259
0 0 3.311 0.154 3.259
0 0 3.311 0.154 3.259

 And I tried to generate a generic EBSD data like this:

`CS = {crystalSymmetry('6/mmm', [2.944 2.944 4.678], 'X||a*', 'Y||b', 'Z||c*', 'mineral', 'Alpha', 'color', 'Red')};`
`ebsd = loadEBSD_generic(fname,'ColumnNames',{'x','y','Euler1','Euler2','Euler3'},'cs',CS);`

But when I do 

`ebsd.orientations.phi1` `ebsd.orientations.Phi` `ebsd.orientations.phi2`

I get different values for the Euler angles:
0.0578 0.0027 0.0569

So maybe i'm doing something wrong when creating this generic EBSD data. All I need is to plot a pole figure from a list of the 3 Euler angles.

Thank you in advance for your help.

Cheers,
Matheus

**Comment by kilir** [comment, marked as answer]
Hi,
you may have to add the option `'radians'`?
See also `help/loadEBSD_generic`.
Hope that helps.
Cheers,
Rüdiger

**Reply by matheusGariglio** [reply]
Hi Rüdiger,

Perfect, thanks!

Cheers,
Matheus
